assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -332,6 +332,124 @@ footer[role="contentinfo"] a {
 
 }
 
+// Alternative form layout 
+.gu-content--contribution-form-two {
+  header[role="banner"] {
+
+    @include mq($from: tablet) {
+      height: 120px;
+    }
+  }
+
+  .guardian-logo {
+    width: 100px;
+
+    svg {
+      width: 100%;
+    }
+
+    @include mq($from: tablet) {
+      width: auto;
+    }
+  }
+
+
+  .gu-content__main {
+    background-color: gu-colour(sport-faded);
+    display: block; // only here to cancel the display: flex
+  }
+
+  .gu-content__content {
+    margin: 0 auto;
+    max-width: 1015px;
+    border: none;
+    padding: 0;
+    background-color: transparent;
+    
+    width: 100%;
+
+    &--flex {
```

Comment:
we don't have a hard and fast rule for this but I generally favor writing full selectors (`gu-content__content--flex` vs `&--flex`) so you can grep them across the project